src/Itinero/Itinero.csproj

```diff
@@ -51,5 +53,9 @@
     <EmbeddedResource Include="Osm\Vehicles\smalltruck.lua">
       <LogicalName>Itinero.Osm.Vehicles.smalltruck.lua</LogicalName>
     </EmbeddedResource>
+    <EmbeddedResource Include="Osm\Vehicles\train.lua" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\reminiscence\src\Reminiscence\Reminiscence.csproj" />
```

Comment:
Same here, local reference

Reply:
yes sorry, I removed the nugets to ensure that I was testing all code against develop, I will restore the nuget and make another pr when I get a chance, I also wanted to add some more tests to reminiscence for parallel access to make sure the issue wasn't actually coming from there.

Reply:
See also https://github.com/OsmSharp/core/pull/101

Reply:
Fixed in latest commit, I changed them back to nugets